src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -152,6 +152,8 @@ std::vector<uint64_t> LiveFunctionsDataView::GetFunctionTimerDurations(int row)
   return timer_durations;
 }
 
+void LiveFunctionsDataView::UpdateHistogram() { UpdateHistogram(GetVisibleSelectedIndices()); }
```

Comment:
Nit: Maybe we don't need both `LiveFunctionsDataView::UpdateHistogram()` and `LiveFunctionsDataView::UpdateHistogram(const std::vector<int>& visible_selected_indices)`. 
I prefer to just keep `LiveFunctionsDataView::UpdateHistogram()` .

Reply:
@karupayun makes the same point. 

Would that be OK to call `GetVisibleSelectedIndices` in `LiveFunctionsDataView::UpdateHistogram()` in  `LiveFunctionsDataView::OnRefresh`? I mean, `visible_selected_indices` are already available, is that 100% OK to re-acquire them?

Reply:
Sorry, I didn't read the previous code carefully. I only didn't like to have two methods unnecessarily. The solution you are proposing is ok but now that I'm reading it, I would rather prefer to have only one method where it includes a function_id as a parameter.

Reply:
I need the one without arguments too -- I call it in `OrbitMainWindow::OnTimerSelectionChanged`

Reply:
Ah, make sense. 

How about changing `LiveFunctionsDataView::UpdateHistogram(const std::vector<int>& visible_selected_indices)` to `LiveFunctionsDataView::UpdateHistogramWithFunctionIds(std::vector<int>& function_ids)`.

Then their names are more clear for different purposes:
`UpdateHistogram` is for updating histogram with visible selections, and `UpdateHistogramWithFunctionIds` is for updating histogram for specified functions.

Reply:
Sorry, github died yesterday and I wasn't able to answer. In OrbitMainWindow::OnTimerSelectionChanged you have function_id, you can use it directly there :)

Reply:
`UpdateHistogram` and `UpdateHistogramWithFunctionIds` make perfect sense to me. But I'd rather have them take a single `int`/`uint64_t`, rather than a vector.

Reply:
As I said before, I would prefer to have only one UpdateHistogram (uint64_t function_id). Makes much more sense in my head providing the function_id to update the histogram. But since the other reviewers already approved the PR, feel free to ignore this comment.

Reply:
OK, I'll do it in another PR to simplify the merge with "Plot histograms faster". I'll add todo for now.